### PR TITLE
Improved work economy with Piscine Mermaid

### PR DESCRIPTION
**Pull request by BLUREMOTE**
## About The Pull Request

Piscine Mermaid now gain 2 Qlipoth Counter when her lover get a good result with her.
Good work result as a lover now also reset the timer for the Qlipoth reduction.
Also added an extra 20 seconds on the timer to make total breach time 10 minutes (up from 9).
Small tweak to her documentation to point out the qlipoth gain along with the timer reset.

## Why It's Good For The Game

With these changes, being the lover of the Mermaid will hog less of your time and make her far more manageable while reducing her impact on the amount of work per meltdowns.
Simply put, if the lover get a good work result, her Qlipoth Counter maxes out AND the Qlipoth Reduction timer of the crown is resetted, meaning you get the maximum possible duration of not working Piscine Mermaid. With the extra 20 seconds added on the timer, you have 3 minute and 20 seconds before a lose of counter and a whole 10 minute before she breaches.
Reducing the amount of time needed to work mermaid over a given amount of time also mean there's more works available as a whole, as you are not dedicating as much works toward mermaid.

### CODE REVIEW

**Comment by ScarSkit** [comment]
I assume its balance, since I read this pr as a buff to piscine mermaid, looks good, especially QoL changes such as a reset on the timer after the "lover" performs good work with the abnormality, and proper up to date documentation

**Review by Coxswain-Navigator — APPROVED** [review]
Makes a bit more sense this way, why not?
